test/integration/regression.spec.js

```diff
@@ -31,7 +31,7 @@ describe('regressions', function() {
       'fixtures',
       'regression',
       '1794',
-      'simple-ui.js'
+      'simple-ui.fixture.js'
```

Comment:
This unrelated change should have been done in separate PR...

Reply:
I felt like that was too much overhead for a rename, given that I'm in the file already.

Reply:
It just makes it that much harder to use GitHub history when all the changes keep happening in unrelated PRs. For simple file rename, assuming checks still work afterwards, I wouldn't care if you had just merged it first.

Anyhow...